backend/coreapp/urls.py

```diff
@@ -9,6 +9,7 @@
     path('scratch/<slug:slug>/compile', views.compile, name='scratch-compile'),
     path('scratch/<slug:slug>/claim', views.ScratchClaim.as_view(), name='scratch-claim'),
     path('scratch/<slug:slug>/fork', views.fork, name='scratch-fork'),
+    path('scratch/<slug:slug>/export', views.export, name='scratch-export'),
```

Comment:
I wonder if it would be better to expose exporting as
```
GET /api/scratch/slug
Accept: application/zip
```
(i.e. export when the requested content-type isnt json)

Reply:
I think that might be difficult to use in practice? 

The way it is - it can be opened with a plain link. I don't think there's a way from JS to tell the browser to open a link with a certain `Accept` header? I think you'd have to XHR the entire data, and then generate a "fake" download link?

Reply:
yeah I'd rather opt for user-friendliness over a more "proper" design

Reply:
@ethteck This is the API, it's not intended to be used by users (and hence correctness is more important to me).

@zbanks good point, nvm. I'm a bit wary of this being under /api though, we can route /scratch/asdf.zip to the API with nextjs I think.